translations/monero-core_it.ts

```diff
@@ -1872,7 +1873,8 @@ Ringsize: </source>
         <location filename="../wizard/WizardPassword.qml" line="127"/>
         <source> &lt;br&gt;Note: this password cannot be recovered. If you forget it then the wallet will have to be restored from its 25 word mnemonic seed.&lt;br/&gt;&lt;br/&gt;
                         &lt;b&gt;Enter a strong password&lt;/b&gt; (using letters, numbers, and/or symbols):</source>
-        <translation type="unfinished"></translation>
+        <translation> &lt;br&gt;Nota: questa password non può essere recuperata. Se l&apos;hai dimenticata, in tal caso il portafoglio può essere ripristinato dal seed mnemonico composto di 25 parole.&lt;br/&gt;&lt;br/&gt;
```

Comment:
`Se l'hai dimenticata` should be present: "se la dimentichi il portafoglio può essere ripristinato[...]"